crates/sdk/src/masp.rs

```diff
@@ -1231,9 +1231,10 @@ impl<U: ShieldedUtils + MaybeSend + MaybeSync> ShieldedContext<U> {
     pub async fn precompute_asset_types<N: Namada>(
         &mut self,
         context: &N,
```

Comment:
Should we just pass in the client at this point, since we don't need the rest of the context?